dotnet/WipeAppWriteBucket/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Net;
+using Appwrite;
```

Comment:
The Appwrite .NET SDK is out of date and does not work with v1 endpoints. Could you please try achieving this task without the SDK by directly sending HTTP requests to the API? This [article](https://dev.to/eldadfux/learn-how-you-can-take-advantage-of-the-appwrite-api-without-using-any-sdk-a41) should help.

Reply:
Done